test/Microsoft.AspNet.Razor.Test/TestFiles/CodeGenerator/Output/AddTagHelperDirective.cs

```diff
@@ -10,13 +10,7 @@ private void @__RazorDesignTimeHelpers__()
         {
             #pragma warning disable 219
             string __tagHelperDirectiveSyntaxHelper = null;
-            __tagHelperDirectiveSyntaxHelper = 
-#line 1 "AddTagHelperDirective.cshtml"
```

Comment:
Line pragmas don't gain us any value during design time. They enabled users to hit breakpoints on `@addTagHelper` etc. directives which is very misleading because they're directives, not in-place methods or instructions. See description for more detail.